### ISSUE TITLE
Talkback Keyboard Controls Android

### ISSUE THREAD

**Issue by dbrezack** [facebook/react-native#20659]
- [x] Review the documentation: https://facebook.github.io/react-native
  - [x] Search for existing issues: https://github.com/facebook/react-native/issues
  - [x] Use the latest React Native release: https://github.com/facebook/react-native/releases

## Environment
Android Version 7.11

```
Environment:
  OS: Linux 4.15
  Node: 8.9.4
  Yarn: 1.5.0
  npm: 5.6.0
  Watchman: Not Found
  Xcode: N/A
  Android Studio: Not Found

Packages: (wanted => installed)
  react: 16.3.1 => 16.3.1
  react-native: ~0.55.2 => 0.55.4
```

## Description
The Talkback click command `alt+enter` does not work on Button or TouchableOpacity elements. I have also tried every combination of the various accessibility props on the button.

It does work with Alerts.

## Reproducible Demo

1. Create new project ```create-react-native-app TestButton```
2. Add a button to the `App.js` file 
```     
 <Button
        onPress={()=> console.warn('It worked')}
        title="Try Me"/>
```
3. Run on either emulator (With play store) or physical device (With external keyboard)
4. Enable Talkback `Settings->Accessibility->Talkback`
5. Navigate back to app
6. Use `alt+enter` to try and click button after navigating to it

**Comment by dbrezack** [comment]
@react-native-bot .56 is not fully supported yet. Still waiting on all the supporting libraries to catch up e.g.(create-react-native-app)

**Comment by B3rry** [comment]
@react-native-bot I can verify that this is still an issue in 0.55.3

**Comment by hramos** [comment]
We're only tracking issues that affect the latest release -- there's too many open issues at the moment, so the minimum we require is that you at least confirm the issue is not present in the latest version.

In this particular case, please do try `master` as we just landed a suite of Accessibility updates (see http://facebook.github.io/react-native/blog/2018/08/13/react-native-accessibility-updates)

**Comment by B3rry** [comment]
I was able to reproduce this in 0.56 using the following code:

<script src="https://gist.github.com/B3rry/43692b208fe610208af86b5e036b7cae.js"></script>

See the video posted here for a live demo: https://twitter.com/JonB3rry/status/1032053258251067392

**Comment by dbrezack** [comment]
Issue is still relevant.

**Comment by adityafuture01** [comment]
Is this issue fixed? Because I still this issue in 0.59 version.

**Comment by renaekathleen** [comment]
I still see this issue in 0.59.3. Is there a good workaround to accommodate users utilizing keyboard navigation?

**Comment by renaekathleen** [comment]
this issue is still relevant.
